src/ucp/api/ucp.h

```diff
@@ -722,9 +721,10 @@ typedef struct ucp_worker_params {
      */
     struct {
         int                 epoll_fd;    /**< epoll file descriptor */
-        epoll_data_t        epoll_data;  /**< data associated with events on
+        uint64_t            epoll_data;  /**< data associated with events on
```

Comment:
is it enough ? kqueue is int. epoll is int. do we have anything else we should care ?

Reply:
epoll is union between int,u32,and u64

Reply:
also i see that for kqueue it is void* (udata):
```
     struct kevent {
	     uintptr_t ident;	     /*	identifier for this event */
	     short     filter;	     /*	filter for event */
	     u_short   flags;	     /*	action flags for kqueue	*/
	     u_int     fflags;	     /*	filter flag value */
	     intptr_t  data;	     /*	filter data value */
	     void      *udata;	     /*	opaque user data identifier */
     };
```

Reply:
I overlooked it...
Let's not call it "epool" ?

Reply:
ah, was looking at another object.